ui-kit-sample/src/main/java/org/p2p/uikit/sample/FinanceBlockFragment.kt

```diff
@@ -0,0 +1,282 @@
+package org.p2p.uikit.sample
+
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.LinearLayoutManager
+import android.os.Bundle
+import android.view.View
+import by.kirich1409.viewbindingdelegate.viewBinding
+import com.hannesdorfmann.adapterdelegates4.AdapterDelegate
+import com.hannesdorfmann.adapterdelegates4.AsyncListDifferDelegationAdapter
+import com.hannesdorfmann.adapterdelegates4.dsl.adapterDelegateViewBinding
+import org.p2p.core.common.IconContainer
+import org.p2p.core.common.TextContainer
+import org.p2p.uikit.components.icon_wrapper.IconWrapperUiModel
+import org.p2p.uikit.components.finance_block.FinanceBlockUiModel
+import org.p2p.uikit.components.finance_block.UiKitFinanceBlockView
+import org.p2p.uikit.components.left_side.LeftSideUiModel
+import org.p2p.uikit.components.right_side.RightSideUiModel
+import org.p2p.uikit.databinding.ItemFinanceBlockBinding
+import org.p2p.uikit.sample.databinding.FragmentFinanceBlockBinding
+import org.p2p.uikit.utils.image.ImageViewUiModel
+import org.p2p.uikit.utils.image.commonCircleImage
+import org.p2p.uikit.utils.text.TextViewBackgroundUiModel
+import org.p2p.uikit.utils.text.TextViewUiModel
+import org.p2p.uikit.utils.toPx
+import kotlin.jvm.internal.Intrinsics
+
+class FinanceBlockFragment : Fragment(R.layout.fragment_finance_block) {
+
+    private val binding by viewBinding(FragmentFinanceBlockBinding::bind)
+
+    private val adapter = Adapter(
```

Comment:
It seems like, when you wanted to integrate compose, but you were the only one with such views )))